recipes/pbcore/1.2.7/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pbcore" %}
+{% set version = "1.2.7" %}
+
+package:
+  name: pbcore
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b3f5feb55007848bd5589b1b48bf586268d1a246dc874cc9759a8dc1b6322a46
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - .open = pbcore.io.opener:entryPoint
```

Comment:
@bishemma1 this looks a bit strange, can it be removed? Otherwise I think this can be merged.

Reply:
Yes I think so, I removed it.